tests/e2e/main.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```

Comment:
You could also add the license info here

Reply:
Done.